pilot/pkg/model/push_context.go

```diff
@@ -1306,6 +1308,9 @@ func (ps *PushContext) updateContext(
 			wasmPluginsChanged = true
 		case kind.EnvoyFilter:
 			envoyFiltersChanged = true
+			if features.OptimizedConfigRebuild {
+				changedEnvoyFilters.Insert(conf)
```

Comment:
Does this work? won't it insert into a nil map?

Reply:
hmm probably doesn't, will initialize set accordingly